apps/backend/app.py

```diff
@@ -185,12 +199,41 @@ def upload_experiment_results(expId, trialExtraFile, postProcess):
     uploadBlob = firebaseBucket.blob(f"results/result{expId}.csv")
     uploadBlob.upload_from_filename('results.csv')
 
+    # Upload to MongoDB
+    try:
+            mongoClient.admin.command('ping')
+    except ConnectionFailure as err:
+        raise GladosInternalError("MongoDB server not available/unreachable") from err
+        
+    
+    print('Uploading to MongoDB')
+    experimentFile = open(f"results.csv") # there is probably a better way to do this
+    experimentData = experimentFile.read()
+    experimentFile.close()
+    experimentResultEntry = {"_id": expId,
+                        "resultContent": experimentData}
+    try:
+        resultId = mongoResultsCollection.insert_one(experimentResultEntry).inserted_id
```

Comment:
```suggestion
        resultId = mongoResultsCollection.insert_one(experimentResultEntry).inserted_id # TODO store this id in the firebase entry somewhere so it can be accessed by the frontend
```

Reply:
ditto